Editor/Window/GameLiftPlugin.cs

```diff
@@ -102,6 +104,9 @@ private void CreateGUI()
             _root.Add(uxml);
 
             ApplyText();
+            
+            _tabContentContainer = _root.Q(className: MainContentClassName);
```

Comment:
I'd prefer it's a parameter since the method is tightly coupled with the existence of the content container property. If we move the definition the method calls will fail. Using a parameter makes this failure obvious reducing complexity when maintaining in the future